### ISSUE TITLE
MIGRATE_PREDEFINED_LABELS may not work in case predefined label's condition is changed. 

### ISSUE THREAD

**Issue by jinfengni** [sundeck-io/OpsCenter#189]
If we change the condition for a predefined label (for instance to fix a bug like #187),  run `MIGRATE_PREDEFINED_LABELS ` will not upgrade the user's label.  

`MIGRATE_PREDEFINED_LABELS` will take effect only when the following requirements are met:
- User does not make any change to existing predefined labels , does not create any user label;
- We add new predefined labels

If we modify an existing predefined label's condition, the following checking will fail 
```
    let rowCount1 number := (
        WITH
        OLD_PREDEFINED_LABELS AS
            (SELECT name, condition, LABEL_CREATED_AT FROM INTERNAL.PREDEFINED_LABELS WHERE TIMESTAMPDIFF(SECOND, LABEL_CREATED_AT, CURRENT_TIMESTAMP) > :gap_in_seconds),
        USER_LABELS AS
            (SELECT name, condition, LABEL_MODIFIED_AT FROM INTERNAL.LABELS)
        SELECT count(*) from (select * from OLD_PREDEFINED_LABELS MINUS SELECT * FROM USER_LABELS) S
        );
    let rowCount2 number := (
        WITH
        OLD_PREDEFINED_LABELS AS
            (SELECT name, condition, LABEL_CREATED_AT FROM INTERNAL.PREDEFINED_LABELS WHERE TIMESTAMPDIFF(SECOND, LABEL_CREATED_AT, CURRENT_TIMESTAMP) > :gap_in_seconds),
        USER_LABELS AS
            (SELECT name, condition, LABEL_MODIFIED_AT FROM INTERNAL.LABELS)
        SELECT count(*) from (select * from USER_LABELS MINUS SELECT * FROM OLD_PREDEFINED_LABELS) S
        );

    IF (rowCount1 > 0 OR rowCount2 > 0) THEN
        RETURN FALSE;
    END IF;

```

**Comment by jinfengni** [comment]
@rymurr ,  is predefined label immutable?  Do we support to modify existing predefined label's condition?

**Comment by jinfengni** [comment]
I put a PR which supports migrating predefined label, if we modify predefined label's condition. 

https://github.com/sundeck-io/OpsCenter/pull/188/commits

**Comment by rymurr** [comment]
closes with #188
